src/usr/local/www/services_dhcp_edit.php

```diff
@@ -468,6 +482,13 @@ function staticmaps_sort($ifgui) {
 	$pconfig['hostname']
 ))->setHelp('Name of the host, without domain part.');
 
+$section->addInput(new Form_Input(
+	'ddnshostname',
+	'DDNS Hostname',
+	'text',
+	$pconfig['ddnshostname']
+))->setHelp('Name of the host to be register in DNS, without domain part.');
```

Comment:
'Name of the host to ~~be~~ register in DNS, without domain part.'
Thanks for contributing.